devito/finite_differences/finite_difference.py

```diff
@@ -182,10 +171,8 @@ def cross_derivative(expr, dims, fd_order, deriv_order, stagger=None, **kwargs):
     >>> f = Function(name='f', grid=grid, space_order=2)
     >>> g = Function(name='g', grid=grid, space_order=2)
     >>> cross_derivative(f*g, dims=(x, y), fd_order=(2, 2), deriv_order=(1, 1))
```

Comment:
hang on: why does this look different now?

Reply:
Some small changes in FD